.vscode/launch.json

```diff
@@ -33,6 +33,16 @@
             "projectName": "ded_card",
             "args": "",
             "envFile": "${workspaceFolder}/.env"
+        },
+        {
+            "type": "java",
```

Comment:
co to za konfiguracja?

Reply:
Nić takiego nie zmieniałem. To chyba mój MAC, widzi i zmienia pliki, które w Windowsie nie są problemem.

Reply:
to jest ewidentnie jakiś twój plugin w vsc